Sources/WishKit/View/WishCell.swift

```diff
@@ -296,41 +296,15 @@ extension WishCell {
             return
         }
 
-        var rootViewController = UIApplication.shared.windows.first(where: \.isKeyWindow)?.rootViewController
-
-        if #available(iOS 15, *) {
-            rootViewController = UIApplication
-                .shared
-                .connectedScenes
-                .compactMap { ($0 as? UIWindowScene)?.keyWindow }
-                .first?
-                .rootViewController
-        }
-
-        if let rootViewController = rootViewController, response.state == .implemented {
-            AlertManager.confirmMessage(on: rootViewController, message: WishKit.config.localization.youCanNotVoteForAnImplementedWish)
```

Comment:
Should this also be printed instead like the others?

Reply:
We do not show implemented wishes, so I thought this case is irrelevant for us.